src/utils/graphql/queries/getReviewInfo.query.ts

```diff
@@ -0,0 +1,32 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getReviewInfo($reviewerId: Int!, $applicationId: Int) {
+    reviewAssignments(
+      condition: { reviewerId: $reviewerId, applicationId: $applicationId }
+      orderBy: TIME_CREATED_DESC
+    ) {
+      nodes {
+        id
+        status
+        timeCreated
+        reviews {
+          nodes {
+            id
+            level
+            status
+            timeCreated
```

Comment:
this was causing an issue

Reply:
we don't seem to set this yet in schema, but definitely should

Reply:
Hi, have you used the back-end branch (as described at the top) as well? And ran `yarn database_init`?

Reply:
oops i didn't sorry